assets/stylesheets/solutions.scss

```diff
@@ -38,3 +38,7 @@
   //border-top: 1px solid #ddd;
   //background-color: #E9FFE0;
 }
+
+aside.quote .title.title-only {
+  padding: 12px;
```

Comment:
[`aside.quote .title { padding: 12px 12px 1px 12px  }` in core.](https://github.com/discourse/discourse/blob/53e6ccf17b48ca216def631cc199af3e2993b412/app/assets/stylesheets/common/base/topic-post.scss#L77)